shared-scripts.sh

```diff
@@ -386,4 +386,12 @@ function emitCanaryFailureMetric {
             --dimensions branch=main \
             --region us-west-2
     fi
+}
+
+function deploy {
+  echo "Deploy"
+  loadCacheFromBuildJob
+  echo "Authenticate with NPM"
+  echo "//registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN" > ~/.npmrc
+  source ./codebuild_specs/scripts/publish.sh
```

Comment:
The script here needs exec instead of `source`? I notice the command in circle is `bash ./.circleci/publish.sh`

Reply:
Fixed this to just use `./`. Thanks